### PR TITLE
9-e Recursive Bayesian filter for linear Gaussian dynamics

### PR DESCRIPTION
**Pull request by aviator2160**
I completely forgot to submit a pull request for this branch. Sorry!

### CODE REVIEW

File: bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+These are handy assumptions because a Gaussian distribution subject to linear dynamics remains Gaussian.
+
+## Preliminaries
+
+### Gaussian random variables
+
+For a random variable $X$, a particular value or sample is represented by the lowercase $x \in \mathbb{R}^n$.
+When we say $X \sim \mathcal{N}(\mu, \Sigma)$, we mean that $X$ is "Gaussian": it follows a Gaussian or normal distribution with mean $\mu \in \mathbb{R}^n$ and covariance $\Sigma \in \mathbb{R}^{n \times n}$:
+$$
+\begin{equation}
```
Comment (ayazaero) [inline bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md]: Equations are not rendering properly. Needs to be corrected.

File: bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+Derive the Kalman filter from the recursive Bayesian filter subject to linear dynamics and Gaussian distributions, and comment on the connection between the two filters.
+
+## Introduction
+
+The recursive Bayesian filter (RBF) is the most general possible statement of how one's belief about the value of a random signal changes over time.
+If one wishes to estimate the state of an uncertain system, the RBF exactly describes how the probability of each state varies over time.
+This is the math that underlies any state estimation problem, from localizing the position of an aircraft using GPS to determining the orientation of a satellite using a star tracker.
+
+However, the RBF is challenging to use in practice because of the difficulty of computing the normalization factor in Bayes' rule, upon which the RBF is based, for systems with many possible states.
```
Comment (ayazaero) [inline bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md]: Can you elaborate on this part? Why is it difficult to compute normalization factor in Bayes' rule?

File: bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
+# Recursive Bayseian Filters for Linear Gaussian Systems
```
Comment (ayazaero) [inline bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md]: Typo in Bayesian

**Review by ayazaero — CHANGES_REQUESTED** [review]
The notes are quite descriptive and useful and brings out the relationship between the recursive Bayesian filter and Kalman filter clearly. Your enthusiasm for this connection is evident.

The formatting of equations is the only major issue which needs immediate attention.

Thanks!

File: bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md (hunk 1 above)
Comment (aviator2160) [reply bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md]: Thanks for catching this! Turns out I just needed to add extra newlines above and below the equation blocks.

File: bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md (hunk 2 above)
Comment (aviator2160) [reply bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md]: Ooh yeah, that's something I should have covered. I've added a paragraph explaining the problem!

File: bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md (hunk 3 above)
Comment (aviator2160) [reply bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md]: Good catch, thank you!

**Pull request merged**